backend/domain/transaction/transaction_crud.py

```diff
@@ -43,6 +43,9 @@ def update_transaction(
     """
     account = get_account_by_id(db, account_id)
```

Comment:
You can also refactor this. It's only two lines but refactoring will make it easier to maintain.

Reply:
how would I refactor this?

Reply:
Bring those two lines out and create a function called "valid_transaction" or something where it checks to make sure that the transaction exists inside the account.